docs/en/reference/configuration.rst

```diff
@@ -295,7 +295,11 @@ pdo\_oci / oci8
 -  ``instancename`` (string): Optional parameter, complete whether to
    add the INSTANCE_NAME parameter in the connection. It is generally used
    to connect to an Oracle RAC server to select the name of a particular instance.
-
+-  ``connectstring`` (string): Complete Easy Connect connection descriptor,
+   see https://docs.oracle.com/database/121/NETAG/naming.htm. When using this option,
+   you will still need to provide the ``user`` and ``password` parameters, but the other
```

Comment:
You messed up some backticks here :D There is missing one after `password` and I suppose `getHost` and `getPort` will require double backticks, too ;) Otherwise fine.